docs/src/1-metabolic-modeling.jl

```diff
@@ -391,6 +392,39 @@ result.exchanges
 
 @test result_with_more_oxygen.exchanges.oxygen < -19.0 #src
 
+# ### Alternative: Using Accessors.jl
+#
+# Accessors.jl implement a "lensy" way to update immutable data structures.
+# That comes with a nice outcome of doing the right amount of shallow copyies
+# for you automatically, thus avoiding much of the technical danger of in-place
+# modifiations. (You still lose the equational reasoning on your code, but that
+# may not be an issue at all in usual codebases.)
+#
+# Accessors interface is used simply through macros `@set` (which sets a deeply
+# nested field in a structure, returning a modified copy), or with `@reset`
+# which automatically "assigns" the result back to the original variable:
+
+using Accessors
+
+c = @set c.exchanges.biomass.bound = C.Between(-50, 50)
+
+# The above code is equivalent to:
+
+@reset c.exchanges.biomass.bound = C.Between(-50, 50)
```

Comment:
Unclear what the difference is

Reply:
it's described 5 lines above, basically "you don't have to `c=` manually".

Should I copypaste or move that here?

Reply:
@stelmo ping ping

Reply:
Ah yeah, on a second reading it was clear :)